src/test/java/net/liquidpineapple/pang/ApplicationTest.java

```diff
@@ -45,6 +48,7 @@ public void testNotInCheatMode() throws Throwable {
 
   @After
   public void tearDown() throws Exception {
+    ScoreSystem.setScore(0);
```

Comment:
jup, because otherwise the score of the scoreSystem is not reset, leading to errors